pkg/controller/jobs/pod/pod_controller.go

```diff
@@ -207,7 +211,7 @@ func (p *Pod) Stop(ctx context.Context, c client.Client, _ []jobframework.PodSet
 	if err == nil {
```

Comment:
If the Pod was deleted at this point, it should also be fine to consider it a success.

Reply:
It is considered a success: https://github.com/kubernetes-sigs/kueue/blob/481ac9d90db3c86de94f1f9176cfd0d605341edc/pkg/controller/jobs/pod/pod_controller.go#L214